schema/deploy/shipping/views.sql

```diff
@@ -131,4 +131,40 @@ create or replace view shipping.flu_assembly_jobs_v1 as
 comment on view shipping.flu_assembly_jobs_v1 is
     'View of flu jobs that still need to be run through the assembly pipeline';
 
+
+create or replace view shipping.return_results_v1 as
+
+    select barcode,
+           case
+             when sample_id is null then 'notReceived'
+             when sample_id is not null and string_agg(present::text, ',') is null then 'processing'
+             when string_agg(present::text, ',') is not null then 'complete'
```

Comment:
`string_agg(present::text, ',') is null` is a little weird to me. What about `count(present) = 0` and `count(present) > 0`?